co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureRegistrationMessage.java

```diff
@@ -23,12 +23,67 @@
  */
 public class InfrastructureRegistrationMessage {
     private String rxMessageIpAddress;
-    private int timeSyncListenPort = 1517; // TODO
-    private int regstratinoListenPort = 1516; // TODO 
+    private String infrastructureId;
+
+    private int rxMessagePort = 1536;
+    private int timeSyncPort = 1517; // TODO
```

Comment:
TODO what? It's good to note what the todo-item is so that somebody else might pick it up later, though generally I like to be careful about cleaning up TODO items before a PR, anything items that remain at that point really need JIRA stories or github issues as they've become a separate work item.